src/core/Akka.FSharp.Tests/ApiTests.fs

```diff
@@ -121,8 +121,8 @@ let ``actor that accepts _ will receive string message`` () =
     response
     |> equals "SomethingToReturn"
 
-[<Fact>]
-// SUCCEEDS
+//[<Fact>]
+// FAILS
```

Comment:
I've discussed about this test with @Horusiath . He told me, that I should remove this test if it is not actual at the moment